packages/tabs/docs/Tabs.stories.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from "react"
```

Comment:
The alignment on the badge doesn't look right
<img width="411" alt="Screen Shot 2021-11-05 at 5 09 41 pm" src="https://user-images.githubusercontent.com/1621182/140466237-f59e8982-8436-463a-9d02-91b733c2f258.png">